eid_config.lua

```diff
@@ -1,3 +1,21 @@
+local Controller = Controller or {}
```

Comment:
I dont like how this is in the config file. But I guess its an ok place for now.

Reply:
yeah it is awkward that something like this isn't in the official API
the enum is taken from Mod Config Menu's inputhelper file; maybe that could be copied into EID's mod_config_menu file, and just linked to like the Keyboard enum is